cassdegrees/templates/registration/login.html

```diff
@@ -0,0 +1,33 @@
+{% extends "base.html" %}
+{% load breadcrumbs %}
+
+{% block page-title %}Login{% endblock %}
+
+{% block subtitle %}CASS Staff Login{% endblock %}
+
+{% block no-breadcrumbs %}{% endblock %}
+
+{% block content %}
+    <form class="anuform" action="" method="post">
+        {% csrf_token %}
+        <fieldset>
+            <legend>Login Details</legend>
+            {# Render form errors #}
+            {% if form.errors %}
+                <br />
+                <div class="msg-error">
+                    {% for x in form.errors.values %}{{ x }}{% endfor %}
+                </div>
+            {% endif %}
+            {# Render form fields #}
+            {% for field in form.visible_fields %}
+                <p>{{ field.label_tag }} {{ field }}</p>
+            {% endfor %}
+        </fieldset>
+
+        <p class="text-right">
+            <a class="left" onclick="alert('Please contact CASS administrators to change your password')">Forgot Password?</a>
```

Comment:
Onclick on a isn't a good idea. Either use href or a `<button>`

Reply:
I'll look in to it, but this seemed like a nice way of quickly notifying the user

Reply:
Na, something like `href="javascript:alert(...)" is what I meant here